tests/snuba/api/endpoints/test_organization_events_stats_mep.py

```diff
@@ -875,13 +873,66 @@ def do_request(self, data, url=None, features=None):
         with self.feature(features):
             return self.client.get(self.url if url is None else url, data=data, format="json")
 
+    def test_top_events_wrong_on_demand_type(self):
+        query = "transaction.duration:>=100"
+        yAxis = ["count()", "count_web_vitals(measurements.lcp, good)"]
+        response = self.do_request(
+            data={
+                "project": self.project.id,
+                "start": iso_format(self.day_ago),
+                "end": iso_format(self.day_ago + timedelta(hours=2)),
+                "interval": "1h",
+                "orderby": ["-count()"],
+                "environment": "production",
+                "query": query,
+                "yAxis": yAxis,
+                "field": [
+                    "count()",
+                ],
+                "topEvents": 5,
+                "dataset": "metrics",
+                "useOnDemandMetrics": "true",
+                "onDemandType": "not_real",
```

Comment:
Is the `onDemandType` a parameter that the UI will be sending? Or is it mostly for us tinkering with the API?